pyscriptjs/src/remote_interpreter.ts

```diff
@@ -152,13 +152,13 @@ export class RemoteInterpreter extends Object {
      * */
     async loadPackage(names: string | string[]): Promise<void> {
         logger.info(`pyodide.loadPackage: ${names.toString()}`);
-        // the new way in v0.22.1 is to pass it as a dict of options i.e.
+        // the new way in v0.22.1 and later is to pass it as a dict of options i.e.
```

Comment:
```suggestion
        // the new way in v0.22.0 and later is to pass it as a dict of options i.e.
```
makes it sound like we made this change in a patch release...

Reply:
Fair point, a leftover from the fact that PyScript jumped from 0.21.3 to 0.22.1.  Adjusted this.